src/nvim/ops.c

```diff
@@ -2538,18 +2538,18 @@ static void yank_copy_line(yankreg_T *reg, struct block_def *bd, size_t y_idx)
 ///
 /// @param oap Operator arguments.
 /// @param reg The yank register used.
-static void yank_do_autocmd(oparg_T *oap, yankreg_T *reg)
+static void do_autocmd_yank(oparg_T *oap, yankreg_T *reg)
   FUNC_ATTR_NONNULL_ALL
 {
-  static bool recursive = false;
+  static bool recursive_yank = false;
```

Comment:
There you go, I just don't know C. I thought that "program-wide lifetime" equals "global namespace", too. :\

I'll change them back.